src/Library/demos/Text Input/main.js

```diff
@@ -0,0 +1,103 @@
+import Adw from "gi://Adw";
+import Gtk from "gi://Gtk";
+
+const entry = workbench.builder.get_object("entry");
+const buffer = entry.buffer;
+const progress_entry = workbench.builder.get_object("progress_entry");
+const icon_entry = workbench.builder.get_object("icon_entry");
+const visibility_entry = workbench.builder.get_object("visibility_entry");
+
+const invert_icons = {
+  "eye-open-negative-filled-symbolic": "eye-not-looking-symbolic",
+  "eye-not-looking-symbolic": "eye-open-negative-filled-symbolic",
+};
+
+entry.connect("activate", () => {
+  console.log(`"${entry.text}" was entered!`);
```

Comment:
```suggestion
  console.log(`"${entry.text}" entered`);
```

Reply:
👍🏾 
Also, I wasn't so sure but do you want me to connect the "activate" signal for all the entries in that section?

Reply:
go for it :+1: